how-to/customize-workspace/client/src/framework/fdc3/1.2/mapper.ts

```diff
@@ -0,0 +1,144 @@
+import type { Image } from "@openfin/workspace";
+import type { PlatformApp } from "customize-workspace/shapes";
+import type {
+	AppInterop,
+	AppIntents as FDC3TwoPointZeroAppIntents
+} from "customize-workspace/shapes/fdc3-2-0-shapes";
+import type { AppDefinition, AppIcon, AppImage, AppIntents, AppMetadata } from "../../shapes/fdc3-1-2-shapes";
+
+function getIcons(icons: AppIcon[]): Image[] {
+	const appIcons: Image[] = [];
+	if (!Array.isArray(icons)) {
+		return appIcons;
+	}
+	for (const appIcon of icons) {
+		appIcons.push({ src: appIcon.icon });
+	}
+	return appIcons;
+}
+
+function getImages(images: AppImage[]): Image[] {
+	const appImages: Image[] = [];
+	if (!Array.isArray(images)) {
+		return appImages;
+	}
+	for (const appImage of images) {
+		appImages.push({ src: appImage.url });
+	}
+	return appImages;
+}
+
+function getManifest(app: AppDefinition): unknown {
+	if (typeof app.manifest === "string" && app.manifest.startsWith("{")) {
+		return JSON.parse(app.manifest);
+	}
+
+	return app.manifest;
+}
+
+function getTags(app: AppDefinition): string[] {
+	// eslint-disable-next-line @typescript-eslint/dot-notation
```

Comment:
Maybe get rid of the eslint and change the getTag signature to `AppDefinition & { tags?: string[] }`